### PR TITLE
feat(ui5-input): introduce openPicker() functionality

### PR DESCRIPTION
**Pull request by stbodurov**
A new `openPicker()` method for manually opening the suggestions popover has been introduced.

Usage:
```js
<ui5-input id="myInput" show-suggestions></ui5-input>
------
myInput.addEventListener("event-type", () => {
	fetch(myRequest)
		.then((response) => {
			// Arrange items before calling openPicker, to prevent an empty popover from showing
			myInput.openPicker();
		});
});
```

A sample implementation with recent item history will be available in [Input with openPicker() on focusin
](https://sap.github.io/ui5-webcomponents/master/playground/main/pages/Input/#openPickerInput) once merged.

### CODE REVIEW

File: packages/main/src/Input.js [hunk 1]
```diff
@@ -1071,6 +1095,7 @@ class Input extends UI5Element {
 
 		this.isTyping = false;
 		this.openOnMobile = false;
+		this._forceOpen = false;
```
Comment (ndeshev) [inline packages/main/src/Input.js:1098]: Is this necessary? It seems that when the suggestion is selected the popover closes and the flag will get reset in _afterClosePopover()

Comment (stbodurov) [reply packages/main/src/Input.js:1098]: Without it being here, the suggestions popover doesn't close when selecting an item.

File: packages/main/test/pages/Input.html [hunk 2]
```diff
@@ -563,6 +566,77 @@ <h3>Inputs with clear icon</h3>
...
+		openPickerInput.addEventListener("focus", handleHistory);
+
+		function handleSuggestions() {
+			fetch('https://dummyjson.com/users?select=firstName,lastName&limit=100')
+				.then(res => res.json())
+				.then(({users}) => {
+					var value = openPickerInput.value;
+					var suggestionItems = [];
+
+					[].slice.call(openPickerInput.children).forEach(child => {
```
Comment (ndeshev) [inline packages/main/test/pages/Input.html:584]: you can also use just [...openPickerInput.children] to convert the HTML collection to an array :)

**Review by ndeshev — APPROVED** [review]

File: packages/main/test/pages/Input.html (hunk 2 above)
Comment (stbodurov) [reply packages/main/test/pages/Input.html:584]: Will keep in mind this tip for the spread operator in the future, thanks.

**Pull request merged**